homeassistant/generated/zeroconf.py

```diff
@@ -0,0 +1,11 @@
+"""Automatically generated by hassfest.
+
+To update, run python3 -m hassfest
+"""
+
+
+SERVICE_TYPES = {
+    "_axis-video._tcp.local.": [
```

Comment:
Let's make it tuples" `("axis",)`

Reply:
oh, that gets a lot more complicated since we're using JSON. It's fine like this.